x-pack/packages/kbn-entities-schema/scripts/generate_oas.js

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+require('../../../../src/setup_node_env');
+
+const { generateOAS } = require('./generate');
```

Comment:
Nit: The file is named `generate` but the export is `generateOAS` while **this** file is named `generate_oas`...